src/ducks/devices.js

```diff
@@ -4,6 +4,20 @@ import update from 'immutability-helper';
 import { keyBy } from 'lodash';
 import { create } from '../lib/redux';
 
+export const {
+  type: ANNOUNCE_SYNC,
+  action: announceSync,
+  reducer: announceSyncReducer
+} = create('ANNOUNCE_SYNC', {
+  success: (state, action) => {
+    const newState = update(state, {
+      $toggle: ['syncAnnounced']
```

Comment:
might not want to put it in the state tree. I think we want to announce every time the view is in focus so we should do it when the isFocused function goes from false to true